src/nft/components/bag/BagRow.tsx

```diff
@@ -39,6 +39,25 @@ const ReviewButton = styled(ThemeButton)`
   padding: 8px;
   width: 50%;
 `
+const RemoveAssetOverlay = styled.div`
```

Comment:
Can you give a demo build of this to product/requester, seems a little to large and currently I can click outside of the card to remove it which doesn't seem to be expected

Reply:
Interesting - what do you mean by "outside of the card"? 

As far as the preview, I've pasted a screenshot of a clickable area on Jira (feel free to check the ticket for reference). I'll ask product to double-check on it too.  

I think the size requirement is largely influenced by Apple:
> Apple’s [iPhone Human Interface Guidelines](https://developer.apple.com/iphone/library/documentation/UserExperience/Conceptual/MobileHIG/DesigningNativeApp/DesigningNativeApp.html#//apple_ref/doc/uid/TP40006556-CH4-SW1) recommends a minimum target size of 44 pixels wide 44 pixels tall. Microsoft’s Windows Phone UI Design and Interaction Guide suggests a touch target size of 34px with a minimum touch target size of 26px. Nokia’s developer guidelines suggest that the target size should be no smaller than 1cm x 1cm square or 28 x 28 pixels.

Reply:
I attached a screenshot in the thread but basically part of the click area is outside of the image

Reply:
Yeah, let's follow up internally and see where we land.